ImgSmall.java

```diff
@@ -0,0 +1,65 @@
+import javax.imageio.ImageIO;
+import java.awt.Image;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Paths;
+
+public class ImgSmall {
+  public static void main(String[] args) throws IOException {
+      resize(args);
+  }
+  public static void resize(String[] args) throws IOException{
+    if (args.length != 2){
+      System.out.println("please put in a url and a percent");
+      System.exit(0);
+    }
+    //get url object
+    URL url = null;
+    try{
+      url = new URL(args[0]);
+    } catch (MalformedURLException mfue){
+      System.out.println("check the url");
+      System.out.print(args[0]);
+      System.exit(0);
+    }
+    double resizeBy = (100 - Integer.parseInt(args[1])) / 100.0;
+    BufferedImage img = null;
+    //get image from url
+
+    img = ImageIO.read(url);
+    if (img == null){
+      System.out.print("Please put in an image");
```

Comment:
The response to the user is not descriptive. You should say "The URL you entered does not contain any image, please check"